src/views/InputView.js

```diff
@@ -21,7 +25,15 @@ class InputView {
         resolve(input);
       });
     });
-  }
-}
+  },
+
+  async inputName() {
+    return InputHandler.getValidInput(MESSAGE.INPUT.NAME, Parser.splitName, Validator.validateInputNames, 'INPUT_NAME');
+  },
+
+  async inputTryNumber() {
+    return InputHandler.getValidInput(MESSAGE.INPUT.TRY_NUMBER, Parser.toNumber, Validator.validateInputTryNumber, 'INPUT_TRY_NUMBER');
```

Comment:
Parser.toNumber 없이 이렇게 해도 괜찮을 것 같아요!

```suggestion
    return InputHandler.getValidInput(MESSAGE.INPUT.TRY_NUMBER, Number, Validator.validateInputTryNumber, 'INPUT_TRY_NUMBER');
```

Reply:
앗 저도 이 코드를 쓸 때, 하는 일이 결국 Number()인데 Parser.toNumber가 굳이 정의되어야할까 고민했습니다. 그 때 별로 더 깊게고민하지 않고, Parser를 쓴다는 통일감을 위해...ㅎㅎ 적었었는데 다시 생각해보니 불필요한 코드였던 것 같습니다!

감사합니다☺️